File: Java Project  V1.7/backend-spring/src/main/java/com/emart/dto/ProductSpecificationDTO.java
package com.emart.dto;

/**
 * Represents a single specification (config name + value)
 * shown on the Product Details page.
 * e.g. { "configName": "RAM", "configValue": "8 GB" }
 */
public class ProductSpecificationDTO {

    private String configName;

    private String configValue;


    // =========================
    // CONSTRUCTORS
    // =========================

    public ProductSpecificationDTO() {
    }

    public ProductSpecificationDTO(
            String configName,
            String configValue) {

        this.configName = configName;
        this.configValue = configValue;
    }


    // =========================
    // GETTERS AND SETTERS
    // =========================

    public String getConfigName() {
        return configName;
    }

    public void setConfigName(String configName) {
        this.configName = configName;
    }


    public String getConfigValue() {
        return configValue;
    }

    public void setConfigValue(String configValue) {
        this.configValue = configValue;
    }
}

File: DotNet Project V2.0/backend-dotnet/src/Emart.Application/Dtos/ProductSpecificationDTO.cs
namespace Emart.Application.Dtos;

public class ProductSpecificationDTO
{
    public string ConfigName { get; set; } = null!;
    public string ConfigValue { get; set; } = null!;

    public ProductSpecificationDTO()
    {
    }

    public ProductSpecificationDTO(string configName, string configValue)
    {
        ConfigName = configName;
        ConfigValue = configValue;
    }
}
